training/src/app/joke/joke.component.ts

```diff
@@ -0,0 +1,16 @@
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+
+@Component({
+  selector: 'app-joke',
+  templateUrl: './joke.component.html',
+  styleUrls: ['./joke.component.scss']
+})
+export class JokeComponent{
+  @Input() joke: string = '';
+  @Output() getJoke : EventEmitter<string> = new EventEmitter<string>();
```

Comment:
why do we need it?

Reply:
it is not part of the demands, I've tried to print message to the console